repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasClassificationDefStoreV2.java

```diff
@@ -83,7 +82,7 @@ public AtlasVertex preCreate(AtlasClassificationDef classificationDef) throws At
         }
         ret = typeDefStore.findTypeVertexByDisplayName(
                 classificationDef.getDisplayName(), TypeCategory.TRAIT);
-        if (ret != null && !classificationDef.getSkipDisplayNameUniquenessCheck()) {
+        if (ret != null && !classificationDef.getAllowDuplicateDisplayName()) {
```

Comment:
refactor here
in case **classificationDef.getAllowDuplicateDisplayName()** is true, no need to call 
        ret = typeDefStore.findTypeVertexByDisplayName(..) since we want allow duplicate names

Reply:
Same goes for line  number 197